src/mjolnir/landmark_database_builder.cc

```diff
@@ -1,8 +1,39 @@
 #include "mjolnir/landmark_database_builder.h"
 #include "mjolnir/util.h"
 
+#include "mjolnir/osmpbfparser.h"
+
 namespace valhalla {
 namespace mjolnir {
+const std::string landmark_db = "landmarks.db";
+
+static const std::unordered_map<std::string, LandmarkType> str_type_map = {
+    {"parking", LandmarkType::parking},
+    {"bench", LandmarkType::bench},
+    {"parking_space", LandmarkType::parking_space},
+    {"place_of_worship", LandmarkType::place_of_worship},
+    {"restaurant", LandmarkType::restaurant},
+    {"waste_basket", LandmarkType::waste_basket},
```

Comment:
yeah no way are we using a trash can to navigate :smile:

Reply:
make sense haha. trash cans can move, not good either.